jscomp/frontend/ast_exp_handle_external.ml

```diff
@@ -69,12 +69,16 @@ let handle_debugger loc (payload : Ast_payload.t) =
       Ast_external_mk.local_external_apply loc ~pval_prim:[ "#debugger" ]
         ~pval_type:(Typ.arrow Nolabel (Typ.any ()) (Ast_literal.type_unit ()))
         [ Ast_literal.val_unit ~loc () ]
-  | _ -> Location.raise_errorf ~loc "bs.debugger does not accept payload"
+  | _ -> Location.raise_errorf ~loc "%%debugger extension does not accept payload"
```

Comment:
An even clearer error message does not assume that the user knows what a payload is.

Reply:
Maybe `%debugger extension don't accept arguments`?